Testnet/NFTStore/NFTStore.Tests/InMemoryState.cs

```diff
@@ -46,7 +47,8 @@ public T GetStruct<T>(string key)
 
         public UInt256 GetUInt256(string key) => GetValue<UInt256>(key);
 
-        public bool IsContract(Address address) => IsContractResult;
+        public void SetIsContract(Address address, bool isContract) => AddOrReplace($"__IsContract:{address}", isContract);
```

Comment:
`private` right?

Reply:
No we want to able to set given address is belong to contract or not on testing stage.

Reply:
"__" prefix is there for avoid key conflicts